Client/Frontend/Browser/OpenSearch.swift

```diff
@@ -65,7 +65,24 @@ class OpenSearchEngine: NSObject, NSCoding {
      * Returns the search URL for the given query.
      */
     func searchURLForQuery(_ query: String) -> URL? {
-        return URL.search(query: query)
+        // Ecosia: Unleash Shortcuts Experiment
+        /* 
+         If the search is performed via Ecosia, we want to follow the standard flow
+         which is now environment-dependant.
+         the function `getURLFromTemplate(::)` takes into account the ecosia search engine
+         which contains only the production URL.
+         Once this experiment is considered finished and the logic needs fallback,
+         leave only the following line of code for the whole function's body
+         
+         `URL.search(query: query)`
+         */
+        if EngineShortcutsExperiment.isEnabled,
+        let fromTemplateURL = getURLFromTemplate(searchTemplate, query: query),
+           fromTemplateURL.baseDomain != "ecosia.org" {
+            return fromTemplateURL
+        } else {
+            return URL.search(query: query)
```

Comment:
`URL.search() is an extension from our `Core` isn't? I like that it's short but it lacks identification with Ecosia. So we could consider renaming it or building the searchURL with a different pattern than an extension. Just food for thought

Reply:
Coolio! Happy to rename it into `URL.ecosiaSearch(query: query)`. Self-explanatory and straightforward. WDYT @ecotopian ?

Reply:
Yes that works!

Reply:
@ecotopian updated the core counterpart and opened a PR with the new function name.